tests/Nelmio/Alice/Fixtures/Builder/Reference.php

```diff
@@ -0,0 +1,232 @@
+<?php
+
+/*
+ * This file is part of the Alice package.
+ *  
+ * (c) Nelmio <hello@nelm.io>
+ *  
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Nelmio\Alice\Fixtures\Builder\Methods;
+
+use Nelmio\Alice\Fixtures\Fixture;
+
+/**
+ * List a list of fixture reference, with the expected result. The list is flagged to be able to easily extract some
+ * parts of it. This class aims at centralising all the possible fixture tested to avoid too much duplication.
+ */
+final class Reference
```

Comment:
@tshelburne I know you're very busy, but I would like you to review that one as it's a fairly big changed. The idea in this class is to provide a full list of possible values which will be used for tests. I'll add comments below to explain the format. There **is BC breaks**. I avoided them at all cost in favour of deprecations, but there is a few of them that are, IMO more bugs. Still, it's my opinion and I may have got a few things wrong, so please do check that one in details.